### ISSUE TITLE
Doubt using i18n-abide client-side translations

### ISSUE THREAD

**Issue by victorgaspar** [mozilla/i18n-abide#109]
Hi,

I do not know how to use i18n-abide for client-side translations. I Googled a lot and checked the documentation, but I couldn't find a clear example.

Basically, I have a client-side javascript message I would like to translate:

`document.getElementById("valorationform").innerHTML = "<h3>Thanks for your feedback!</h3><br><br>You can now close this window.";`

I tried including gettext.js in my html page, using <%, gettext function, etc. but I think I do not understand properly how i18n-abide works for client-side. Server-side messages are working like a charm.

Could you please give me an example on how to use i18n-abide to translate the above text? Including any required includes, etc.

Thanks in advance.

**Comment by shane-tomlinson** [comment]
Hi @victorgaspar - i18n-abide is a Express middleware (for Node) and is not meant to be used client-side within the browser. Unfortunately I can't really point you in a good direction for tools I've used myself, on the project I currently work on we use homegrown tooling that is straight up awful. Mozilla does have a great localization team that is working on a robust client side library that may be of use to you: https://github.com/l20n/l20n.js

**Comment by victorgaspar** [comment]
Hi @shane-tomlinson,
thanks for your message. I probably misunderstood the documentation:
  https://github.com/mozilla/i18n-abide
where it says: 

> This module supports client side as well as server side localization.

I also got confused with the gettext.js:
https://github.com/mozilla/i18n-abide/commit/d3b48c1e0fad9f94274ccd9d8f5f7ca56c2486c1

I thought it would be used for client side translations:

> Have client side strings and you’ll include a gettext.js script in your code. This is distributed with i18n-abide.

Nevertheless, I implemented a small workaround. It is not nice, but at least I do not depend in yet another library. I just added a script var in my HTML page:
`<script type="text/javascript">var msgThanks = "<%= gettext('Thanks for your feedback!') %>";</script>`

and then I use it from my JS:
`document.getElementById("valorationform").innerHTML = "<h3>" + msgThanks + "</h3>"`

Not nice for a very complex page full of messages, but for a small page with few messages it will do.

Regards,
Victor
